.travis.yml

```diff
@@ -20,6 +20,33 @@ matrix:
       python: 3.5
     - os: linux
       python: 2.7
+    - os: linux
+      # The PROJECT environment variable refers to a downstream project that
+      # depends on cloudpickle (for example: distributed, joblib, loky). For
+      # each project, a matrix item is created, that will run the $PROJECT
+      # test suite with the latest cloudpickle, to check for breaking changes
+      # introduced by cloudpickle.
+      # Side note: for distributed, the pytest major version is constrained to
+      # 3 because running it's test suite with pytest 4 fails for now. Also,
+      # two failing tests related to openssl and not to cloudpickle are
+      # skipped.
+      python: 3.7
+      sudo: required
+      env: PROJECT=distributed
+           TEST_REQUIREMENTS="pytest==3.6 numpy pandas mock bokeh"
+           PROJECT_URL=https://github.com/dask/distributed.git
+           PYTEST_ARGS="-k not test_connection_args and not test_listen_args"
+      if: commit_message =~ /(ci-downstream|ci-distributed)/
```

Comment:
To be consistent with the `[ci skip]` [convention](https://circleci.com/docs/2.0/skip-build/) maybe we should rename this to `[ci downstream]` / `[ci distributed]`. I think the following might work:


```yaml
      if: commit_message =~ /(\[ci downstream\]|\[ci distributed\])/
```

although I am not 100% sure how escaping works in regexp in travis.yml.